src/rgw/rgw_rados.cc

```diff
@@ -12128,6 +12128,10 @@ int RGWRados::update_containers_stats(map<string, RGWBucketEnt>& m)
         ent.size_rounded += stats.total_size_rounded;
       }
     }
+
+    // fill in placement_rule from the bucket instance, in case we didn't write
+    // one in rgw_link_bucket()
```

Comment:
my understanding is, we won't write one in rgw_link_bucket(), with following commit?

Reply:
updated, thanks